framework/include/ble_manager/ble_manager.h

```diff
@@ -0,0 +1,97 @@
+/****************************************************************************
+ *
+ * Copyright 2021 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/**
+ * @defgroup BLE_Manager BLE_Manager
+ * @ingroup BLE_Manager
+ * @brief Provides APIs for BLE Manager
+ * @{
+ */
+
+/**
+ * @file ble_manager/ble_manager.h
+ * @brief Provides APIs for BLE Manager
+ */
+
+#ifndef BLE_MANAGER_H_
+#define BLE_MANAGER_H_
+
+#include "ble_common.h"
+#include "ble_error.h"
+#include "ble_client.h"
+#include "ble_server.h"
+
+typedef enum {
+	BLEMGR_SERVER_MODE,
+	BLEMGR_CLIENT_MODE,
+	BLEMGR_COMBO_MODE,
+	BLEMGR_UNKNOWN_MODE,
+} blemgr_mode_e;
+
+typedef struct _blemgr_context {
+	blemgr_mode_e mode;
+	// conn_id
+} blemgr_context;
+
+/****************************************************************************
+ * Name: ble_manager_init
+ *
+ * Description:
+ *   Initialize ble manager.
+ * 
+ * Input Parameters:
+ *   client_config - Client config. If this is NULL, diable client mode.
+ *   server_config - Server config. If this is NULL, diable server mode.
+ *
+ * Returned Value
+ *   Zero (BLE_RESULT_SUCCESS) is returned on success; a negated value is returned on
```

Comment:
For `ble_result` type, positive value is returned on failure.
Please check and modify the description for every APIs.

Reply:
Yes, you are right.  I will fix 'negated' to 'positive'